src/libs/reportUtils.js

```diff
@@ -25,11 +25,11 @@ function getReportParticipantsTitle(logins) {
 /**
  * Check whether a report action is Attachment is not.
  *
- * @param {Object} reportMessageText report action's message as text
+ * @param {Object} reportActionMessage report action's message as text and html
```

Comment:
Also, I feel like we need a more sophisticated comment now. we should tell why are we ignoring the message when HTML is `[Attachment]`.

Reply:
Does this sound good?
```diff
/**
 * Check whether a report action is Attachment or not.
+ * If both text and html of reportActionMessage is [Attachment], text '[Attachment]' not an attachment is the message.
 *
 * @param {Object} reportActionMessage report action's message as text and html
 * @returns {Boolean}
 */
```

Reply:
Ignore messages containing `[Attachment]` as the main content. Attachments are actions with only text as `[Attachment]`.